packages/flutter/ios/breez_sdk_liquid.podspec

```diff
@@ -0,0 +1,24 @@
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
+# Run `pod lib lint breez_sdk.podspec` to validate before publishing.
+Pod::Spec.new do |s|
+  s.name                   = 'BreezSDKLiquid'
+  s.version                = '0.2.1'
```

Comment:
version script should update the version on this podspec.

Reply:
Done [00bfede](https://github.com/breez/breez-sdk-liquid/pull/488/commits/00bfede5f84fbd40e966b444db74a14a88fa0695)